integrationtests/tests/fixtures/legacy.rs

```diff
@@ -0,0 +1,419 @@
+use std::error::Error as StdError;
```

Comment:
Looks like a lot of code, but it's mostly just delegation to implement the new interfaces.